src/components/tables/QueueTable.tsx

```diff
@@ -365,12 +380,29 @@ const QueueTable = () => {
       {({ handleArchive, hasAdminSelected, isArchiving }) => (
         <>
           <BaseTable
+            actionItems={
+              <>
+                <Button
+                  size="sm"
+                  gap={2}
+                  colorScheme="orange"
+                  variant="outline"
+                  onClick={openContributeModal}
+                >
+                  Contribute
```

Comment:
I'm not sure but "contribute" sounds vague to me. How about "Add a transcript"?
@kouloumos what do you think?